mer/run.py

```diff
@@ -24,21 +24,52 @@ def convert_dbl_to_dict(ref_dbl, rec_dbl):
     return examples
 
 
+def convert_txt_to_dict(txt):
+    examples = txt.read()
+    ref_rec_pairs = examples.split("\n\n")
+
+    examples = []
+    for item in ref_rec_pairs:
+        try:
+            ref, rec, targets, reason = item.split("\n")
+        except:
+            print(item)
+        _, ref = ref.split(":")
+        _, rec = rec.split(":")
+        _, targets = targets.split(":")
+        _, reason = reason.split(":")
+
+        example = {"reference": ref, "recognised": rec, "reason": reason, "minor": 0, "standard": 0, "serious": 0}
+
+        for word in targets.split():
+            example[word] += 1
+
+        examples.append(example)
+
+    return examples
+
+
 def main():
 
     parser = argparse.ArgumentParser()
     # pylint: disable=line-too-long
     # fmt: off
-    parser.add_argument("--ref_dbl", type=argparse.FileType("r"), required=True, help="Dbl file containing paths to reference transcripts")  # noqa:  E201
-    parser.add_argument("--rec_dbl", type=argparse.FileType("r"), required=True, help="Dbl file containing paths to recognised transcripts")  # noqa:  E201
+    inputs = parser.add_mutually_exclusive_group()
+    inputs.add_argument("--example_path", type=argparse.FileType("r"), required=False, help="txt file containing reference, recognised pairs for processing")
+    dbls = inputs.add_argument_group()
+    dbls.add_argument("--ref_dbl", type=argparse.FileType("r"), required=False, help="Dbl file containing paths to reference transcripts")  # noqa:  E201
+    dbls.add_argument("--rec_dbl", type=argparse.FileType("r"), required=False, help="Dbl file containing paths to recognised transcripts")  # noqa:  E201
     parser.add_argument("--prompt_config_path", type=str, default="./config/prompt.json", help="path to prompt config json")  # noqa:  E201
     parser.add_argument("--output_json", type=str, default="./results_dbl.json", help="path to output json to store results")  # noqa:  E201
     parser.add_argument("--api_key", type=str, default=None, help="api key for open ai")  # noqa:  E201
     parser.add_argument("--num_samples", type=str, default=3, help="number of times to sample GPT3 for majority voting")  # noqa:  E201
     # fmt: on
     args = parser.parse_args()
 
-    examples = convert_dbl_to_dict(args.ref_dbl, args.rec_dbl)
+    if args.example_path:
+        examples = convert_txt_to_dict(args.example_path)
+    else:
+        examples = convert_dbl_to_dict(args.ref_dbl, args.rec_dbl)
```

Comment:
don't use run.py, use test.py as that expects an input json rather than dbls

Reply:
done